case-repository/src/main/java/eu/itesla_project/cases/EntsoeAndXmlCaseRepository.java

```diff
@@ -37,7 +37,7 @@ public EntsoeAndXmlCaseRepository(EntsoeCaseRepositoryConfig config, Computation
         super(config,
                 Arrays.asList(new EntsoeFormat(Importers.getImporter("CIM1", computationManager), "CIM"),
                         new EntsoeFormat(Importers.getImporter("UCTE", computationManager), "UCT"), // official ENTSOE formats)
-                        new EntsoeFormat(Importers.getImporter("XML", computationManager), "XML")), // IIDM XML format
+                        new EntsoeFormat(Importers.getImporter("XML", computationManager), "IIDM")), // IIDM XML format
```

Comment:
Importers.getImporter("IIDM"), I guess, would be more accurate in this case ( XMLImporter's and XMLExporter's getFormat have to be changed, accordingly - together with all their occurrences)
However that would imply iidm=xml   What about the json iidm format mentioned in the issue 17?  what would it be its "format name"?
I wonder if Importers.getImporter("XIIDM") could be a better future-proof, option  .....  
Besides , what about changing classes names EntsoeAndXmlCaseRepository to EntsoeAndIidmCaseRepository ?
What do you think?